src/compare/index.ts

```diff
@@ -25,26 +25,26 @@ export async function resolveCompare(provider: IHistoryProvider, opts: StorageOp
 }
 
 export async function resolvePrevBenchmark(
-  compareWith: CompareWith,
+  compareWith: CompareWithType,
   provider: IHistoryProvider
 ): Promise<Benchmark | null> {
   switch (compareWith.type) {
-    case CompareWithType.exactCommit:
+    case CompareWith.exactCommit:
       return await provider.readHistoryCommit(compareWith.commitSha);
 
-    case CompareWithType.latestCommitInBranch: {
+    case CompareWith.latestCommitInBranch: {
       // Try first latest commit in branch
       return await provider.readLatestInBranch(compareWith.branch);
     }
   }
 }
 
-export function renderCompareWith(compareWith: CompareWith): string {
+export function renderCompareWith(compareWith: CompareWithType): string {
```

Comment:
just noting here that while adding a suffix to the type is ok, I don't think using `Type` as suffix is ideal since we use that for ssz types already, so maybe `Options` / `Opts` might be better?

Reply:
the pattern would still be the same, just different suffix, see comment from NC, he voted for option 4 but voiced something similar here https://github.com/ChainSafe/lodestar/discussions/7574#discussioncomment-12504512

we are probably fine with using `Type` as suffix, just wanna point this out so we consider it at least

Reply:
I am fine with either suffix, as long as everyone is aligned with it.

Reply:
Please no `Options`.... its a "type"....